SortMeRNA/4.2.0/SortMeRNA.nf

```diff
@@ -31,6 +31,8 @@ process SortMeRNA {
             gzip -f < non-rRNA-reads.fastq > ${fastq_files[0].simpleName}_non_rRNA.fastq.gz
             gzip -f < rRNA-reads.fastq > ${fastq_files[0].simpleName}_filtered_rRNA.fastq.gz
             mv rRNA-reads.log ${report_title}_rRNA_report.txt
+            rm non-rRNA-reads.fastq
+            rm rRNA-reads.fastq
```

Comment:
Would it not be possible to use the `zip-out` option, like you use in 4.3.3? Assuming this results the .gz output.

Reply:
zip-out is only introduced in 4.3.1 (first update in a year since 4.2.0) so not an option unfortunately

Reply:
bummer :D